torchfields/voting.py

```diff
@@ -27,15 +27,24 @@ def gaussian_blur(self, sigma=1, kernel_size=5):
     kernel = kernel.expand(2, 1, *kernel.shape)
     return F.conv2d(padded, weight=kernel, groups=2)
 
+def get_vote_shape(self):
+    """Consistently split shape into N fields & shape of field
+    """
+    n, _, *shape = self.shape
+    return n, shape
 
-def vote(self, softmin_temp=1, blur_sigma=1):
-    """Produce a single, consensus displacement field from a batch of
-    displacement fields
+def get_vote_subsets(self):
+    """Compute list of majority subsets to use in vote
```

Comment:
Add a short statement here and in the other new functions saying that this function is used as part of vector voting, and to see the vote function for details, or something along those lines.

Since these will now be exposed as core methods of the field object, it makes sense to explain to users what they're for.